release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -1229,6 +1229,8 @@ This script removes unauthenticated subjects from the following cluster role bin
 +
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1821771[*BZ#1821771*])
 
+* If you use a persistent volume (PV) for Prometheus when running monitoring in your cluster, you might experience OOM kills during the upgrade to {product-title} 4.7. Prometheus memory usage doubles during cluster upgrade when a PV is in use. To avoid the OOM kill issue, allow nodes with double the size of memory that was available prior to the upgrade. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1925061[*BZ#1925061*])
```

Comment:
* Is this redundant? Can you run monitoring without Prometheus or vice-versa?
_Prometheus when running monitoring_

* s/when a PV is in use/when it uses a PV ?? (I am assuming that a PV associated with another component doesn't cause the problem)

* Something is missing here:
_allow nodes with double the size of memory that was available prior to the upgrade_ 

_increase memory to allow the nodes to double..._ or something like that.

Reply:
@mburke5678 Thank you for your questions. @simonpasquier @lilic @s-urbaniak, can you please confirm if my answers are accurate?
> * Is this redundant? Can you run monitoring without Prometheus or vice-versa?
>   _Prometheus when running monitoring_
Prometheus is a default component in the monitoring stack. It is not a requirement to use a PV for storage of Prometheus data. But if you are using a PV for Prometheus, then you can experience this issue of OOM kill during upgrade. 

> * s/when a PV is in use/when it uses a PV ?? (I am assuming that a PV associated with another component doesn't cause the problem)
Yes, the PV in this case is specifically for Prometheus, which is why the previous sentence says "when running monitoring." But the second sentence can be more clear: "When a PV is in use for Prometheus, Prometheus memory use doubles during cluster upgrade."
> * Something is missing here:
>   _allow nodes with double the size of memory that was available prior to the upgrade_
>   Maybe?
>   _increase memory to allow the nodes to double..._ or something like that.
I think we're trying to specify that you have to double the capacity based on what is being used prior to upgrade. That amount depends on usage for your individual cluster, so we're trying to be general here.
>   Do we need to state that you can revert the memory changes after upgrade?
Sur, Simon, Lili -- Do you think we should state how long you need this increased memory capacity? Ser, I think you said 2 weeks in our meeting on this issue.

Reply:
Made updates based on peer review feedback.